### PR TITLE
thefuck: update to 3.32.

### PR DESCRIPTION
**Pull request by glaulher**
#### Testing the changes
- I tested the changes in this PR: **briefly**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64)
- I built this PR locally for these architectures (if supported. mark crossbuilds):  
  - armv7l

### CODE REVIEW

File: srcpkgs/thefuck/template [hunk 1]
```diff
@@ -12,12 +12,11 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="MIT"
 homepage="https://github.com/nvbn/thefuck"
 distfiles="https://github.com/nvbn/thefuck/archive/${version}.tar.gz"
-checksum=ac12705b95e22b356f23b20181a34d549267b7b1702523a1f0a22ba7ba3d30df
+checksum=76cbb014473672d1c384922857f8fbc1f6f7774f74f784149ad88751854ecfdf
 
 case "$XBPS_TARGET_MACHINE" in
 	# archs where go is available - the test suite uses it
 	x86_64*|i686*|armv[67]*|aarch64*|ppc64le*) checkdepends+=" go" ;;
-	*) make_check=no ;;
```
Comment (paper42) [inline srcpkgs/thefuck/template:20]: Could you explain why you removed this line?

Comment (glaulher) [reply srcpkgs/thefuck/template:20]: It was giving error in lint and there was no error in the tests on my computer.

Comment (paper42) [reply srcpkgs/thefuck/template:20]: > It was giving error in lint 

```
srcpkgs/thefuck/template:20: explain why the tests fail
```
then you should explain why the tests fail on this line (or right before it).

This is a case statement and your PC probably matches the first case, so make_check=no isn't set for you. Restore this line and add something like `... ;; # go is not available`

Comment (glaulher) [reply srcpkgs/thefuck/template:20]: really, thanks

**Pull request merged**